Resources/Prototypes/GameRules/subgamemodes.yml

```
@@ -83,7 +83,10 @@
   - type: XenoborgsRule
   - type: RuleGrids
   - type: GameRule
-    minPlayers: 40
+    # Start Harmony specific change
+    minPlayers: 35
```

```suggestion
    minPlayers: 35 # 40 -> 35
```
Comment the old value too